tools/packaging/common/envoy_bootstrap.json

```diff
@@ -99,6 +99,18 @@
       {
         "regex": "(wasm_filter\\.(.+?)\\.)",
         "tag_name": "wasm_filter"
+      },
+      {
+        "tag_name": "authz_enforce_result",
```

Comment:
Same here, if other stats somehow includes `allowed` `denied`, it will be accidently extracted as `authz_enforce_result` label. Can we make this regex more specific?

Reply:
I tried to use `\\.rbac(\\.(allowed|denied))` to only match stats from `rbac` filter but I found a strange behavior that after I put the `rbac` in front the first capture group in the regex, only one tag extractor will be included for a stat at a time.

The final result is that I will get 2 stats with different tags, one stat with the tag `authz_dry_run_action` and the other with the tag `authz_dry_run_result`.

Instead I want the two tags to appear in the same stat, which is the current result if I put everything inside the first capture group.

I'm not sure if I'm doing it wrong or if it is supposed to be so, please let me know if you have any ideas about it.

Reply:
Could you paste the full stats string here? should be at `/stats` page.

Reply:
Please see the stats output here: https://gist.github.com/yangminzhu/e613695c8e95de18ace5934702fc44cd

Reply:
updated.